docs/tools/firmware.sh

```diff
@@ -103,7 +103,7 @@ create_deb () {
 }
 
 create_rpm () {
-	if [ ! -f "/usr/local/bin/rpmbuild" ]
+	if [ ! command -v  rpmbuild &> /dev/null ]
```

Comment:
`&> /dev/null` is supported only by bash 4 or higher iirc. macOS uses bash 3.

Have you tested this script yourself?

Reply:
Il revert this t0o, I had rpmbuild installed so I never saw the stderr output.